### PR TITLE
Add 404 page #49

### PR DESCRIPTION
**Pull request by ginnaaceror**
### Explication
Add the view for the 404 error. The user will be redirected to this view when requesting a resource that does not exist

### Screenshots
image

- [x] The image and text remains centered
- [x] The component is responsive

### CODE REVIEW

**Review by Israel-Laguan — COMMENTED** [review]
Please remove the change in package-lock.json

```sh
git checkout dev
git fetch -a && git pull
git checkout feature/#49Add-404-page
git checkout origin/dev -- package-lock.json
git commit -m "fix: remove extra change"
git push origin feature/#49Add-404-page
```

File: src/pages/NotFound.jsx [hunk 1]
```diff
@@ -1,6 +1,21 @@
+import notFound from '../assets/not-found.svg'
```
Comment (Israel-Laguan) [inline src/pages/NotFound.jsx]: Name should start with Uppercase and show is an image, i.e. NotFoundSVG

File: src/pages/NotFound.jsx [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+import styled from 'styled-components'
+import { Text } from '../components'
+
+const NotFoundImg = styled.img`
+  display:block;
+  margin:auto;
+  width: 70%;
+  height: 70%;
```
Comment (Israel-Laguan) [inline src/pages/NotFound.jsx]: Did you verified on mobile? Please check and add media query if needed, I think in Mobile is OK to use width: 100%

Comment (Israel-Laguan) [reply src/pages/NotFound.jsx]: Can you center the content i mobile, please?
image
yes, I already check

Comment (ginnaaceror) [reply src/pages/NotFound.jsx]: > Did you verified on mobile? Please check and add media query if needed, I think in Mobile is OK to use width: 100%

for me it looks good on mobile too and with 100% the image remains with a scrolling effect.

Comment (Israel-Laguan) [reply src/pages/NotFound.jsx]: Granted, can you center vertically, please?

**Comment by denisosuna** [comment]
Hi @ginnaaceror by adding a ` margin-top: 20vh;` for mobile it would do the job

image
image

**Comment by ginnaaceror** [comment]
> margin-top: 20vh;

Hi, @denisosuna thanks for you feedback, but I have 2 questions:

Add this property to what element and what would it be solving?
I ask because at this moment I see it centered and it is responsive

| Mobile view | Desktop View |
| :-: | --- |
| ![mobile-preview] | ![desktop-preview] |

[mobile-preview]: https://user-images.githubusercontent.com/76055458/137479101-f942a96c-c843-4d85-92e6-17fd772a3f40.png

[desktop-preview]: https://user-images.githubusercontent.com/76055458/137479263-a0f26302-6dea-43ec-ba56-5687a239b3b0.png

**Review by Israel-Laguan — APPROVED** [review]
I confirm and looks centered as expected!

I removed extra lockfile and lint the code

**Pull request merged**